File: scripts/maritime/inspect_excel.js
const XLSX = require('xlsx');

const filePath = '/home/astrodev/Desktop/logistica-yadran/public/Trazabilidad Estructuras CY.xlsx';

try {
    const workbook = XLSX.readFile(filePath);
    const sheetName = workbook.SheetNames[0];
    const worksheet = workbook.Sheets[sheetName];
    const data = XLSX.utils.sheet_to_json(worksheet);

    console.log(JSON.stringify({
        columns: data.length > 0 ? Object.keys(data[0]) : [],
        sample: data.slice(0, 5)
    }, null, 2));
} catch (error) {
    console.error('Error reading Excel:', error.message);
}

File: scripts/maritime/inspect_excel.py
import pandas as pd
import json

file_path = '/home/astrodev/Desktop/logistica-yadran/public/Trazabilidad Estructuras CY.xlsx'
try:
    df = pd.read_excel(file_path)
    # Get column names and first 5 rows
    result = {
        "columns": df.columns.tolist(),
        "sample": df.head(5).to_dict(orient='records')
    }
    print(json.dumps(result, indent=2))
except Exception as e:
    print(f"Error: {e}")
